app/models/concerns/early_career_payments_helpable.rb

```diff
@@ -36,4 +36,12 @@ def no_entire_term_contract?
   def trainee_teacher?
     nqt_in_academic_year_after_itt == false
   end
+
+  def qualification_name
```

Comment:
We don't need to worry about this now, and my module naming wasn't great here, but I planned to delete this entire module. Currently, it's where the methods are for the implementation of the ineligible screens. That whole lot is going to be replaced when I tackle the updated ineligible screens. We could keep this module around but it'll literally only have your new method in it.

Is there anywhere else your new method could go?

Reply:
When you guys had a go at this, you coped the method from the ECP to the LUP eligibility model: https://github.com/DFE-Digital/claim-additional-payments-for-teaching/compare/master...capt-518-degree-question-response-in-teacher-journey#diff-d5a4e998dcde35819d9253160606c2aebbedcab062cdf7e8306e51172fdf142fR77. I just looked for a module that both eligibilities have access to so I don't have duplication. Happy to move it somewhere else. Or have the duplication if you think that works better.

Reply:
Would `app/helpers/early_career_payments_helper.rb` be alright?